pkg/pipelines/argocd/argocd_test.go

```diff
@@ -70,7 +77,10 @@ func TestBuildCreatesArgoCD(t *testing.T) {
 			TypeMeta:   applicationTypeMeta,
 			ObjectMeta: meta.ObjectMeta(meta.NamespacedName(ArgoCDNamespace, "test-dev-http-api")),
 			Spec: argoappv1.ApplicationSpec{
-				Source: *makeAppSource(testEnv, testEnv.Apps[0], testRepoURL),
+				Source: argoappv1.ApplicationSource{
+					RepoURL: testRepoURL,
+					Path:    filepath.Join(config.PathForApplication(testEnv, testApp), "overlays"),
```

Comment:
I'd probably have simplified both of these to using the same pattern, but this works for me.